includes/class-wc-checkout.php

```diff
@@ -299,7 +299,9 @@ public function create_order( $data ) {
 					$order->{"set_{$key}"}( $value );
 
 				// Store custom fields prefixed with wither shipping_ or billing_. This is for backwards compatibility with 2.6.x.
-				} elseif ( 0 === stripos( $key, 'billing_' ) || 0 === stripos( $key, 'shipping_' ) ) {
+				// TODO: Fix conditional to only include shipping/billing address fields in a smarter way without str(i)pos.
```

Comment:
This TODO wasn't done now because it'd be a breaking change:
```
Boro Sitnikovski [6:07 PM] 
`stripos` is tricky. it'd be better if we relied on https://github.com/woocommerce/woocommerce/blob/master/includes/class-wc-order.php#L51-L74 instead (edited)
but I know we use `str(i)pos` all across the place

Mike Jolley [6:08 PM] 
Depends if 3rd parties are using this as a hack..
```